### ISSUE TITLE
search parsers are broken

### ISSUE THREAD

**Issue by alberanid** [alberanid/imdbpy#136]
#### Issue description

Search parsers are broken.
Test suite output:

```
========================================================================== FAILURES ==========================================================================
_____________________________________________ test_found_many_result_should_contain_correct_number_of_companies ______________________________________________

search_company = <function search_company.<locals>.retrieve at 0x7fc17331a950>

    @mark.fragile
    def test_found_many_result_should_contain_correct_number_of_companies(search_company):
        page = search_company('pixar')
        data = parser.parse(page)['data']
>       assert len(data) >= 38
E       assert 0 >= 38
E        +  where 0 = len([])

tests/test_http_search_company.py:22: AssertionError
___________________________________________________ test_found_one_result_should_be_list_with_one_keyword ____________________________________________________

search_keyword = <function search_keyword.<locals>.retrieve at 0x7fc17317c1e0>

    @mark.fragile
    def test_found_one_result_should_be_list_with_one_keyword(search_keyword):
        page = search_keyword('zoolander')
        data = parser.parse(page)['data']
>       assert data == ['colander']
E       AssertionError: assert [] == ['colander']
E         Right contains more items, first extra item: 'colander'
E         Use -v to get the full diff

tests/test_http_search_keyword.py:22: AssertionError
______________________________________________ test_found_many_result_should_contain_correct_number_of_keywords ______________________________________________

search_keyword = <function search_keyword.<locals>.retrieve at 0x7fc17317c1e0>

    @mark.fragile
    def test_found_many_result_should_contain_correct_number_of_keywords(search_keyword):
        page = search_keyword('messiah')
        data = parser.parse(page)['data']
>       assert 40 < len(data) < 50
E       assert 40 < 0
E        +  where 0 = len([])

tests/test_http_search_keyword.py:29: AssertionError
___________________________________________________ test_found_too_many_result_should_contain_200_keywords ___________________________________________________

search_keyword = <function search_keyword.<locals>.retrieve at 0x7fc17317c1e0>

    def test_found_too_many_result_should_contain_200_keywords(search_keyword):
        page = search_keyword('computer')
        data = parser.parse(page)['data']
>       assert len(data) == 200
E       assert 0 == 200
E        +  where 0 = len([])

tests/test_http_search_keyword.py:35: AssertionError
____________________________________________________ test_found_one_result_should_be_list_with_one_movie _____________________________________________________

search_movie = <function search_movie.<locals>.retrieve at 0x7fc173307950>

    def test_found_one_result_should_be_list_with_one_movie(search_movie):
        page = search_movie('od instituta do proizvodnje')
        data = parser.parse(page)['data']
>       assert data == [
            ('0483758', {'kind': 'short', 'title': 'Od instituta do proizvodnje', 'year': 1971})
        ]
E       AssertionError: assert [] == [('0483758', {'kind': 'short...roizvodnje', 'year': 1971})]
E         Right contains more items, first extra item: ('0483758', {'kind': 'short', 'title': 'Od instituta do proizvodnje', 'year': 1971})
E         Use -v to get the full diff

tests/test_http_search_movie.py:21: AssertionError
_______________________________________________ test_found_many_result_should_contain_correct_number_of_movies _______________________________________________

search_movie = <function search_movie.<locals>.retrieve at 0x7fc173307950>

    @mark.fragile
    def test_found_many_result_should_contain_correct_number_of_movies(search_movie):
        page = search_movie('ace in the hole')
        data = parser.parse(page)['data']
>       assert 185 < len(data) < 200
E       assert 185 < 0
E        +  where 0 = len([])

tests/test_http_search_movie.py:30: AssertionError
____________________________________________________ test_found_too_many_result_should_contain_200_movies ____________________________________________________

search_movie = <function search_movie.<locals>.retrieve at 0x7fc173307950>

    def test_found_too_many_result_should_contain_200_movies(search_movie):
        page = search_movie('matrix')
        data = parser.parse(page)['data']
>       assert len(data) == 200
E       assert 0 == 200
E        +  where 0 = len([])

tests/test_http_search_movie.py:36: AssertionError
____________________________________________________ test_found_many_result_should_contain_correct_movie _____________________________________________________

search_movie = <function search_movie.<locals>.retrieve at 0x7fc173307950>

    def test_found_many_result_should_contain_correct_movie(search_movie):
        page = search_movie('matrix')
        data = parser.parse(page)['data']
        movies = dict(data)
>       assert movies['0133093'] == {'title': 'The Matrix', 'kind': 'movie', 'year': 1999}
E       KeyError: '0133093'

tests/test_http_search_movie.py:43: KeyError
_____________________________________________________________ test_found_movie_should_have_kind ______________________________________________________________

search_movie = <function search_movie.<locals>.retrieve at 0x7fc173307950>

    def test_found_movie_should_have_kind(search_movie):
        page = search_movie('matrix')
        data = parser.parse(page)['data']
        movies = dict(data)
>       assert movies['0106062'] == {'title': 'Matrix', 'kind': 'tv series', 'year': 1993}
E       KeyError: '0106062'

tests/test_http_search_movie.py:50: KeyError
__________________________________________________________ test_found_movie_should_have_imdb_index ___________________________________________________________

search_movie = <function search_movie.<locals>.retrieve at 0x7fc173307950>

    def test_found_movie_should_have_imdb_index(search_movie):
        page = search_movie('blink')
        data = parser.parse(page)['data']
        movies = dict(data)
>       assert movies['4790262'] == {'title': 'Blink', 'imdbIndex': 'IV',
                                     'kind': 'movie', 'year': 2015}
E       KeyError: '4790262'

tests/test_http_search_movie.py:57: KeyError
_______________________________________________ test_found_many_result_should_contain_correct_number_of_movies _______________________________________________

search_movie_keyword = <function search_movie_keyword.<locals>.retrieve at 0x7fc1731d2b70>

    @mark.fragile
    def test_found_many_result_should_contain_correct_number_of_movies(search_movie_keyword):
        page = search_movie_keyword('colander')
        data = parser.parse(page)['data']
>       assert len(data) == 4
E       assert 0 == 4
E        +  where 0 = len([])

tests/test_http_search_movie_keyword.py:22: AssertionError
____________________________________________________ test_found_too_many_result_should_contain_50_movies _____________________________________________________

search_movie_keyword = <function search_movie_keyword.<locals>.retrieve at 0x7fc1731d2b70>

    def test_found_too_many_result_should_contain_50_movies(search_movie_keyword):
        page = search_movie_keyword('computer')
        data = parser.parse(page)['data']
>       assert len(data) == 50
E       assert 0 == 50
E        +  where 0 = len([])

tests/test_http_search_movie_keyword.py:28: AssertionError
____________________________________________________ test_found_many_result_should_contain_correct_movie _____________________________________________________

search_movie_keyword = <function search_movie_keyword.<locals>.retrieve at 0x7fc1731d2b70>

    def test_found_many_result_should_contain_correct_movie(search_movie_keyword):
        page = search_movie_keyword('colander')
        data = parser.parse(page)['data']
        movies = dict(data)
>       assert movies['0382932'] == {'title': 'Ratatouille', 'kind': 'movie', 'year': 2007}
E       KeyError: '0382932'

tests/test_http_search_movie_keyword.py:35: KeyError
_______________________________________________ test_found_many_result_should_contain_correct_number_of_people _______________________________________________

search_person = <function search_person.<locals>.retrieve at 0x7fc174478598>

    @mark.fragile
    def test_found_many_result_should_contain_correct_number_of_people(search_person):
        page = search_person('keanu reeves')
        data = parser.parse(page)['data']
>       assert len(data) == 4
E       assert 0 == 4
E        +  where 0 = len([])

tests/test_http_search_person.py:22: AssertionError
===================================================== 14 failed, 136 passed, 6 skipped in 156.02 seconds =====================================================

```

**Comment by alberanid** [comment]
fixed in master; we'll probably have a 6.4 release soon.

**Comment by alberanid** [comment]
fixed releasing version 6.4
